packages/databyss-api/src/routes/api/helpers/pagesHelper.js

```diff
@@ -73,3 +75,192 @@ export const composeBlockList = list => {
   })
   return result
 }
+
+const replacePatch = async (p, req) => {
+  const _prop = p.path[0]
+  switch (_prop) {
+    case 'entityCache': {
+      await modelDict(p.value.type).findOneAndUpdate(
```

Comment:
Let's use `model.save()` to update instead of `findOneAndUpdate` so we can share more code